services/apis/drs/drsQuestions.js

```diff
@@ -23,6 +23,16 @@ module.exports = {
    */
   fileEquals(res, fileNode) {
     const object = res.data;
+
+    console.log('---------------------drs----------------------');
+    console.log(res);
+    console.log('---------------------fileNode----------------------');
+    console.log(fileNode);
+    console.log('---------------------drs.checksum----------------------');
+    console.log(object.checksums);
```

Comment:
We forgot to add timestamps to correlate the intermittent `undefined` entries with the fence/indexd/revproxy logs 😓